python/src/wslink/backends/aiohttp/relay.py

```diff
@@ -88,14 +88,14 @@ async def send(self, msg):
             elif msg.type == aiohttp.WSMsgType.CLOSE:
                 await self._ws.close()
             else:
-                print("Invalid message to forward")
+                logger.error("Invalid message to forward")
         else:
-            print("client::send - NO SEND")
-            print(f"{self._connected} - {self._ws.closed}")
-            print("-" * 60)
+            logger.error("client::send - NO SEND")
+            logger.error(f"{self._connected} - {self._ws.closed}")
```

Comment:
maybe use better logger infrastructure to prevent string generation when not printing them...